GRPC/courierServer.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "GRPC/courierProto";
+
+package courier;
+
+service CourierServer {
+  rpc CreateOrder(OrderCourierServer) returns (google.protobuf.Empty) {}
+  rpc GetDeliveryService(google.protobuf.Empty) returns (ServiceResponse) {}
+}
+
+message OrderCourierServer{
+  int64 OrderID = 1;
+  int64 CourierServiceID = 2;
+  string RestaurantAddress = 3;
+  string RestaurantName = 4;
```

Comment:
why do you pass name and address of the restaurant instead of referring it by id?

Reply:
In the structure of the order table, a field with a name and address is provided, this is also necessary due to the different types of id for services, for food delivery it is uuid and for the courier of the service everything is through id int.